src/frontend/src/components/createnewproject/UploadArea.tsx

```diff
@@ -101,6 +102,56 @@ const UploadArea = ({ flag, geojsonFile, setGeojsonFile }) => {
     dispatch(CreateProjectActions.SetTotalAreaSelection(null));
   };
 
+  useEffect(() => {
+    function isWGS84GeoJSONBasedOnCoordinates(geojson) {
```

Comment:
@NSUWAL123 can you keep this function Seperately to a utils folder functions file not here as it can be used in multiple places